UnityProject/Assets/Scripts/Systems/FilthGenerator/FilthGenerator.cs

```diff
@@ -72,14 +72,20 @@ public void RunFilthGenerator()
 				}
 			}
 
-			int numberOfTiles = (int) ((EmptyTiled.Count / 100f) * filthDensityPercentage);
+			int numberOfPlayers = PlayerList.Instance.AllPlayers.Count;
+			float densityPercentagePerPlayer = 1f / 2f; // 1% per 2 players
```

Comment:
why make a literal division here instead of just assigning the result of that known operation?

Reply:
thought it was clearer this way tbh

Reply:
well 1 / 2 is 0.5

I'd say this is clearer.
```cs
float densityPercentagePerPlayer = 0.5
```

Reply:
also the comment is a lie

Reply:
technically it isn't a lie, because after the calculation it *is* 1% per two players since (int) will round up

however, I'll remove it just incase someone gets confused about it